Makefile

```diff
@@ -461,10 +461,11 @@ Protos/generated_swift_names_enum_cases.proto: Protos/mined_words.txt
 	@echo '// Swift compile errors are probably bugs in protoc-gen-swift' >> $@
 	@echo 'syntax = "proto3";' >> $@
 	@echo 'package protobuf_unittest_generated;' >> $@
-	@echo 'enum GeneratedSwiftReservedEnum {' >> $@
-	@echo '  NONE = 0;' >> $@
-	@cat Protos/mined_words.txt | awk 'BEGIN{n = 1} {print "  " $$1 " = " n ";"; n += 1 }' >> $@
-	@echo '}' >> $@
+	@#echo 'enum GeneratedSwiftReservedEnum {' >> $@
```

Comment:
We should have a really clear comment here explaining why these lines are commented out and what needs to be done to un-comment them.

Reply:
Oops, meant to do that, thanks, added.